api/api.proto

```diff
@@ -100,6 +100,26 @@ message VersionResp {
   int32 api = 2;
 }
 
+// RefreshTokenRef contains the metadata for a refresh token that is managed by the storage.
+message RefreshTokenRef {
+  // ID of the refresh token.
+  string id = 1;
+  string client_id = 2;
+  string created_at = 3;
+  string last_used = 4;
+}
+
+// ListRefreshReq is a request to enumerate the refresh tokens of a user.
+message ListRefreshReq {
+  string user_id = 1;
```

Comment:
per IRL conversation the users of this API wont have the user_id and conn_id. only the subject from the id_token. Right now the subject is just the user_id[0].

Just like we did for refresh tokens, we need to embed more information in the id token's subject. We need to add the conn_id.

I'm going to advocate that we make another internal struct like we did for refresh tokens[1], then mimic the serialization.[2]

[0] https://github.com/coreos/dex/blob/v2.1.0/server/oauth2.go#L270
[1] https://github.com/coreos/dex/blob/v2.1.0/server/internal/types.proto
[2] https://github.com/coreos/dex/blob/v2.1.0/server/handlers.go#L656-L664